src/integer_mod_q/poly_over_zq/from.rs

```diff
@@ -85,6 +165,45 @@ impl<Mod: Into<Modulus>> From<(&PolyOverZ, Mod)> for PolyOverZq {
     }
 }
 
+impl<Integer: Into<Z>, Mod: Into<Modulus>> From<(Integer, Mod)> for PolyOverZq {
+    /// Creates a [`PolyOverZq`] from a [`PolyOverZ`] and [`Modulus`].
+    ///
+    /// Parameters:
+    /// - `poly`: the coefficients of the polynomial.
+    /// - `modulus`: the modulus by which each entry is reduced.
+    ///
+    /// # Examples:
+    /// ```
+    /// use qfall_math::integer_mod_q::{PolyOverZq, Modulus};
+    /// use qfall_math::integer::PolyOverZ;
+    /// use std::str::FromStr;
+    ///
+    /// let poly = PolyOverZ::from_str("4  0 1 102 3").unwrap();
+    /// let modulus = Modulus::from(100);
+    ///
+    /// let mod_poly = PolyOverZq::from((&poly, &modulus));
+    ///
+    /// # let cmp_poly = PolyOverZq::from_str("4  0 1 2 3 mod 100").unwrap();
+    /// # assert_eq!(cmp_poly, mod_poly);
+    /// ```
```

Comment:
```suggestion
    /// Creates a [`PolyOverZq`] from any values that implement [`Into<Z>`] and [`Into<Modulus>`],
    /// where the second value must be bigger than `1`.
    ///
    /// Parameters:
    /// - `z`: the single, constant coefficient of the polynomial.
    /// - `modulus`: the modulus by which each entry is reduced.
    ///
    /// # Examples:
    /// ```
    /// use qfall_math::integer_mod_q::PolyOverZq;
    /// # use std::str::FromStr;
    ///
    /// let mod_poly = PolyOverZq::from((5, 42));
    ///
    /// # let cmp_poly = PolyOverZq::from_str("5 mod 42").unwrap();
    /// # assert_eq!(cmp_poly, mod_poly);
    /// ```
```
You just forgot to change this example.